### PR TITLE
Update readme page

### PR DESCRIPTION
**Pull request by Imranr2**
Closes https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/22

Changes:

- Update Ui.png
- Update contents to match project
- Update the link of the GitHub Actions build status badge (Build Status) so that it reflects the build status of team repo.
- Acknowledge original source

### CODE REVIEW

**Comment by EuniceSim142** [comment]
LGTM!

**Comment by sivayogasubramanian** [comment]
Please link the issue to your PR description and update the issue accordingly

File: docs/index.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2122S1-CS2103T-W16-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W16-2/tp/actions)
+[![codecov](https://codecov.io/gh/AY2122S1-CS2103T-W16-2/tp/branch/master/graph/badge.svg?token=PQVA8CSXQ8)](https://codecov.io/gh/AY2122S1-CS2103T-W16-2/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**BobTheBodyBuilder(BBTB) is a desktop application for gym managers to manage their clients and gym bookings.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```
Comment (sivayogasubramanian) [inline docs/index.md:11]: Can we change this to align it with https://github.com/AY2122S1-CS2103T-W16-2/tp/pull/17/files?

Comment (Imranr2) [reply docs/index.md:11]: Yup. Will make the change

**Review by sivayogasubramanian — APPROVED** [review]
LGTM 🎉 !

**Pull request merged**